class.jetpack-twitter-cards.php

```diff
@@ -36,6 +36,18 @@ static function twitter_cards_tags( $og_tags ) {
 		}
 
 		if ( ! is_singular() || ! empty( $og_tags['twitter:card'] ) ) {
+			/**
+			 * Filter the default Twitter card image, used when no image can be found in a post.
+			 *
+			 * @since 5.8
```

Comment:
We've passed the code freeze period for 5.8.0. Could you update this to `5.9.0`?